src/components/HashButton.tsx

```diff
@@ -13,12 +13,51 @@ import {grey} from "../themes/colors/aptosColorPalette";
 import ChevronRightRoundedIcon from "@mui/icons-material/ChevronRightRounded";
 import ChevronLeftRoundedIcon from "@mui/icons-material/ChevronLeftRounded";
 import {truncateAddress} from "../pages/utils";
+import {assertNever} from "../utils";
+
+export enum HashType {
+  ACCOUNT = "account",
+  TRANSACTION = "transaction",
+  OTHERS = "others",
+}
+
+function HashLink(hash: string, type: HashType): JSX.Element {
+  switch (type) {
+    case HashType.ACCOUNT:
+      return (
+        <Link
+          component={RRD.Link}
+          to={`/account/${hash}`}
+          target="_blank"
+          color="inherit"
+        >
+          {hash}
+        </Link>
+      );
+    case HashType.TRANSACTION:
+      return (
+        <Link
+          component={RRD.Link}
+          to={`/txn/${hash}`}
+          target="_blank"
+          color="inherit"
+        >
+          {hash}
+        </Link>
+      );
+    case HashType.OTHERS:
```

Comment:
is it being used somewhere?

Reply:
Not yet but I figured it's good to have it